src/ert/gui/ertwidgets/ensembleselector.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         update_ert: bool = True,
         show_only_initialized: bool = False,
         show_only_undefined: bool = False,
+        show_only_no_children: bool = False,
```

Comment:
Should we add a comment describing what this does?
I giess we can infer from usage but not trivial I think.